src/CONST.js

```diff
@@ -179,7 +179,7 @@ const CONST = {
     PRIVACY_URL: 'https://use.expensify.com/privacy',
     LICENSES_URL: 'https://use.expensify.com/licenses',
     PLAY_STORE_URL: 'https://play.google.com/store/apps/details?id=com.expensify.chat&hl=en',
-    ADD_SECONDARY_LOGIN_URL: 'settings?param={%22section%22:%22account%22}',
+    ADD_SECONDARY_LOGIN_URL: 'settings?param=%7B%22section%22:%22account%22,%22openModal%22:%22secondaryLogin%22%7D',
```

Comment:
I think we can make use of EncodeURIComponent instead of hard coding this.

Reply:
I tried that but for some reason it either opens a blank screen or the account page without the modal open. Maybe I'm missing something?

Reply:
Nvm I think I figure out what I did wrong.

Reply:
But safari should support these. Did you try enocdeURI?

like `encodeURI(settings?param=JSON.stringify({section: 'account', openModal:'secondaryLogi'})` OR
like `encodeURIComponent(settings?param=JSON.stringify({section: 'account', openModal:'secondaryLogi'})`

Reply:
Yea, `encodeURI` works! Thanks!